internal/aws/ecsutil/testdata/task_metadata.json

```diff
@@ -0,0 +1,109 @@
+{
```

Comment:
Is there a copy of this file left over that we can delete and reference this one instead?

Reply:
I removed the initial from the ecs container metrics receiver and am using the broken out rest client test helper.